src/assembly/extra/LICENSE

```diff
@@ -460,3 +460,51 @@ Copyright 2017 The TensorFlow Authors.  All rights reserved.
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+===============================================================================
+
+The following compile-scope dependencies come under the MIT License
+
+JCuda (jcuda.org)
+
+org.jcuda.jcuda:0.8.0
```

Comment:
Please update these to reflect maven groupId:artifactId:version, so `org.jcuda.jcuda:0.8.0` is `org.jcuda:jcuda:0.8.0`, etc.